scripts/auto-rebase/rebase_patches/0030-kube-controller-manager-command-context.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go b/vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go
+index 28caa49e..9045347f 100644
+--- a/vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go
```

Comment:
What are the chances of being able to upstream a change so we don't need to continue to carry this patch?

Reply:
OK, I was thinking k/k. If this isn't really useful in o/k it probably makes as much sense to carry it here as there. I don't have a preference.